packages/dev/core/src/Cameras/Inputs/BaseCameraMouseWheelInput.ts

```diff
@@ -59,14 +61,24 @@ export abstract class BaseCameraMouseWheelInput implements ICameraInput<Camera>
     public attachControl(noPreventDefault?: boolean): void {
         noPreventDefault = Tools.BackCompatCameraNoPreventDefault(arguments);
 
-        this._wheel = (pointer) => {
-            // sanity check - this should be a PointerWheel event.
-            if (pointer.type !== PointerEventTypes.POINTERWHEEL) {
-                return;
+        this._connectedObserver = this.camera._deviceSourceManager!.onDeviceConnectedObservable.add((deviceSource) => {

```

Comment:
`_deviceSourceManager!`

I'm a bit confused with this. Is it always there or not? Lots of code use this, but some use `_deviceSourceManager?`. Which is it?